docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,414 @@
-
-To list all existing spaces:
+2. Enter the following statement to create the _team_ tag:
 
 ```ngql
-nebula> SHOW SPACES;
+nebula> CREATE TAG team (name string);
 ```
 
-To create a new space named `test`:
+3. Enter the following statement to create the _follow_ edge type:
 
 ```ngql
-nebula> CREATE SPACE test(partition_num=1024, replica_factor=1);
--- Use this space
-nebula> USE test;
+nebula> CREATE EDGE follow(degree int);
 ```
 
-* `replica_factor` specifies the number of replicas in the cluster.
-* `partition_num` specifies the number of partitions in one replica.
-
-### Define Graph Schema
-
-The `CREATE TAG` statement defines a tag, with a type name and an attribute list.
+4. Enter the following statement to create the _serve_ edge type:
 
 ```ngql
-nebula> CREATE TAG course (name string, credits int);
-nebula> CREATE TAG building (name string);
-nebula> CREATE TAG student (name string, age int, gender string);
+nebula> CREATE EDGE serve(start_year int, end_year int);
 ```
 
-The `CREATE EDGE` statement defines an edge type.
+Now you can check the tags and edge types you just created.
+
+To show the tags you just created, enter the following statement:
 
 ```ngql
-nebula> CREATE EDGE follow(likeness double);
-nebula> CREATE EDGE choose(grade int);
+nebula> SHOW TAGS;
 ```
 
-To list the tags and edge types that we just created：
+The following information is displayed:
 
-```ngql
--- Show tag list
-nebula> SHOW TAGS;
+```
 ============
 | Name     |
 ============
-| building |
+| player   |
 ------------
-| course   |
+| team     |
 ------------
-| student  |
-------------
--- Show edge type list
+```
+
+To show the edge types you just created, enter the following statement:
+
+```ngql
 nebula> SHOW EDGES;
+```
+
+The following information is displayed:
+
+```
 ==========
 | Name   |
 ==========
-| choose |
+| serve |
 ----------
 | follow |
 ----------
 ```
 
-To show the attributes of a tag or an edge type:
+To show the attributes of the _player_ tag, enter the following statement:
 
 ```ngql
--- Show attributes of a tag
-nebula> DESCRIBE TAG student;
+nebula> DESCRIBE TAG player;
+```
+
+The following information is displayed:
+
+```
 ===================
 | Field  | Type   |
 ===================
 | name   | string |
 -------------------
 | age    | int    |
 -------------------
-| gender | string |
--------------------
--- Show attributes of an edge type
+```
+
+To show the attributes of the _follow_ edge type, enter the following statement:
+
+```ngql
 nebula> DESCRIBE EDGE follow;
+```
+
+The following information is displayed:
+
+```
 =====================
 | Field    | Type   |
 =====================
-| likeness | double |
+| degree   | int    |
 ---------------------
 ```
 
-### Insert Data
+### Inserting Data
+
+You can insert vertices and edges based on relations in the illustration figure.
+
+#### Inserting Vertices
+
+You can insert some vertices by the following statements:
```

Comment:
give a few explanation of vertexid (vid for short)

The vertexid should be unique in the space.